single-local-recipe-build.yml

```diff
@@ -40,7 +40,7 @@ parameters:
     default: ''
 
 steps:
-- script: ${{ parameters.python }} -m pip install --upgrade conan
+- script: ${{ parameters.python }} -m pip install conan==1.59.0
```

Comment:
is there anyway to not have hardcoded version? like have a parameter with default version so it can be overwritten?